src/disp_s1/browse_image.py

```diff
@@ -1,15 +1,98 @@
 """Module for creating browse images for the output product."""
 from __future__ import annotations
 
+import argparse
+
+import h5netcdf
 import numpy as np
+import scipy
 from dolphin._types import Filename
 from numpy.typing import ArrayLike
 from PIL import Image
 
 
```

Comment:
Wait how about we actually do put the names up here
```suggestion
    DATASET_CHOINCES = [
        "unwrapped_phase",
        "connected_component_labels",
        "temporal_coherence",
        "interferometric_correlation",
        "persistent_scatterer_mask",
    ]
```


So that in the argparser, we have
```python
   parser.add_argument(
        "-n", "--dataset-name", choices=DATASET_CHOICES, help="Name of dataset to plot from NetCDF file"
    )
```